esdb/impl.go

```diff
@@ -161,6 +161,19 @@ func (handle *connectionHandle) SupportsFeature(feature int) bool {
 	return false
 }
 
+func (handle *connectionHandle) GetServerVersion() (ServerVersion, error) {
+	if handle.Id() == uuid.Nil {
+		return ServerVersion{}, fmt.Errorf("connection is not established")
```

Comment:
Considering that function is public, I would prefer we use the esdb.Error type instead when returning an error.

Reply:
fixed, thanks!